types/swagger-schema-official/index.d.ts

```diff
@@ -41,7 +41,7 @@ export interface Header extends BaseSchema {
 
 // ----------------------------- Parameter -----------------------------------
 
-export type ParameterType = 'string' | 'number' | 'integer' | 'boolean' | 'array' | 'object';
+export type ParameterType = 'string' | 'number' | 'integer' | 'boolean' | 'array' | 'object' | 'file';
```

Comment:
Why is this required? This is a BaseType and we're already handling the exception in https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/swagger-schema-official/index.d.ts#L103

Reply:
Because `Response.schema : Schema : BaseSchema` and therefore `Response.schema.type: ParameterType`.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/cbc8a1ec78414e0131adddd72fb44574358d156e/types/swagger-schema-official/index.d.ts#L147

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/cbc8a1ec78414e0131adddd72fb44574358d156e/types/swagger-schema-official/index.d.ts#L154

And from https://swagger.io/docs/specification/2-0/describing-responses/ :

> ### Response Body
> The schema keyword is used to describe the response body. A schema can define:
> - object or array – typically used with JSON and XML APIs,
> - a primitive such as a number or string – used for plain text responses,
> - file ...